pkg/cedar/intake/models/easi_biz_case.go

```diff
@@ -1,11 +1,11 @@
 package models
 
 // NOTE: these types are used to create a schema used by the CEDAR Intake API
-// When changing any of these types, update the biz case version in pkg/cedar/intake/translation/constants.go (IntakeInputSchemaEASIBizCaseVersion)
+// When changing any of these types, add a new version for BizCase in pkg/cedar/intake/translation/constants.go
```

Comment:
This change undoes work from https://github.com/CMSgov/easi-app/pull/1668; please manually reverse it.